src/test/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZUtilsTest.java

```diff
@@ -0,0 +1,613 @@
+/*
+ * Copyright (c) 2021 AVI-SPL Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.communicator.aver.ptz;
+
+import static com.avispl.symphony.dal.communicator.aver.ptz.AverPTZUtils.buildSendPacket;
+import static com.avispl.symphony.dal.communicator.aver.ptz.AverPTZUtils.convertOneByteNumberToTwoBytesArray;
+import static org.junit.Assert.assertArrayEquals;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import org.junit.Test;
+
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.Category;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.PayloadType;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.command.Command;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.command.CommandType;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.AEMode;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.BGainControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.BacklightStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.FocusControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.FocusStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.PanTiltDrive;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.PowerStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.PresetControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.RGainControl;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.SlowPanTiltStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.SlowShutterStatus;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.WBMode;
+import com.avispl.symphony.dal.communicator.aver.ptz.enums.payload.param.ZoomControl;
+
+/**
+ * Unit test for AverPTZ Utils - method build send string
+ * Build send string which match expected string
+ *
+ * @author Harry
+ * @version 1.0
+ * @since 1.0
+ */
+public class AverPTZUtilsTest {
+	int cameraID = 1;
+	int sequenceNumber = 1;
```

Comment:
what happen if you only use the same sequence number here?
Do we really need to keep it up to date? Since UT can run concurrency or in some condition, this number will impact to the expected result.